packages/storybook-builder-vite/svelte/csf-plugin.js

```diff
@@ -33,13 +33,19 @@ module.exports = {
                 '// export default '
             );
 
+            const namedExportsOrder = Object.entries(stories)
+              .filter(([, def]) => !def.template)
+              .map(([id]) => id)
+              .join("', '");
+
             const output = [
                 codeWithoutDefaultExport,
                 `import parser from '${parser}';`,
                 `const __storiesMetaData = parser(${component}, ${JSON.stringify(
                     all
                 )});`,
                 'export default __storiesMetaData.meta;',
+                `export const __namedExportsOrder = ['${namedExportsOrder}'];`,
```

Comment:
A simpler option is to just
```
`export const __namedExportsOrder = ${JSON.stringify(namedExportsOrder)};`
```

And then `namedExportsOrder` can simply be an array of strings? There's virtually zero runtime overhead, and saves you from some potential formatting mistakes (theoretically speaking), but most of all makes the code a bit easier to read (in my opinion)?

Reply:
Yup agree, this is currently just a copy and paste from https://github.com/modernweb-dev/web/blob/6c5893cc79c91e82f9aec35d3011c6e33ce878cc/packages/dev-server-storybook/src/shared/stories/injectExportsOrder.ts